k8s/bases/common/gridcapa-interruption-server-configmap.yaml

```diff
@@ -0,0 +1,15 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: interruption-server
```

Comment:
Almost all configmap names end with "-configmap". Rao-runner's configmap is an exception.
You should apply the rule here.